src/backend/tests/projects.test.ts

```diff
@@ -917,19 +917,13 @@ describe('Projects', () => {
       );
     });
 
-    test('Deleted Material Tye does not work if the material type is already deleted', async () => {
-      vi.spyOn(prisma.material_Type, 'findUnique').mockResolvedValue({ ...toolMaterial, dateDeleted: new Date() });
-      await expect(ProjectsService.deleteMaterialType('NERSoftwareTools', batman)).rejects.toThrow(
```

Comment:
we want to do toolmaterials.name instead of the NERSoftwareTools. Do this in the other cases of this as well

Reply:
do this on like 909 as well